bookkeeper-server/src/main/java/org/apache/bookkeeper/util/DoubleByteBuf.java

```diff
@@ -280,6 +281,56 @@ public int setBytes(int index, ScatteringByteChannel in, int length) throws IOEx
     }
 
     @Override
+    protected short _getShortLE(int index) {
+        throw new UnsupportedOperationException();
```

Comment:
It's not required since we're using the `DoubleByteBuf` for parsing any data. It's just used to aggregate 2 buffers (typically headers and payload) and write them together on the socket channel. 

This same changes were already merged in BK 4.5.0: https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/util/DoubleByteBuf.java#L277